src/lib.rs

```diff
@@ -92,14 +88,14 @@ impl Launch {
             node_cmd.push_arg(keep_alive_interval_msec.to_string());
         }
 
-        if self.is_local {
+        if self.common.is_local {
             node_cmd.push_arg("--skip-igd");
         }
```

Comment:
A small nitpick is that this condition could move into `CommonArgs::node_cmd`, which would avoid repeating it in both `Launch` and `Join`. Not a blocking issue though.